great_expectations/expectations/core/expect_table_columns_to_match_ordered_list.py

```diff
@@ -34,6 +34,9 @@ class ExpectTableColumnsToMatchOrderedList(BatchExpectation):
     expect_table_columns_to_match_ordered_list is a \
     [Batch Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations).
 
+    BatchExpectations are one of the most common types of Expectation.
+    They are evaluated for an entire Batch, and answer a semantic question about the Batch itself.
```

Comment:
```suggestion
    They evaluate an entire Batch, and answer a semantic question about the Batch.
```
Apply the edit to all similar statements.